manage_user_page.php

```diff
@@ -185,14 +194,49 @@
 $t_where_params = array();
 if( $f_filter === 'ALL' ) {
 	$t_where = '(1 = 1)';
+} else if( mb_substr( $f_filter, 0, 6 ) === 'SEARCH' ) {
+	$t_pos = mb_strpos( $f_findname, '-' );
```

Comment:
I did not actually test the code (no time, sorry) but I think this will not work as expected when searching for a string containing a `-` e.g. `jean-pierre`

Reply:
Fair point the '-' character is used to indicate start of exclusion string. Do you have a suggestion for another character?

Reply:
I have no time to have a deeper look at the moment but adding a space might work
`$t_pos = mb_strpos( $f_findname, ' -' );`

Reply:
Would it be acceptable to use ' -'  (so space & '-')?

Reply:
Have tested ' -' and that works. Since I cannot make a change here, below what it should be:
	$t_pos = strpos($f_findname, ' -');
	if( $t_pos ) {
		$t_exclude = trim( substr( $f_findname,$t_pos+2 ) );
		$f_findname = trim( substr( $f_findname, 0,$t_pos ) );

Reply:
This is not very robust. What if I search for `-xxx` ?